### DISCUSSION TITLE
Assistance Needed for Memory Usage and Disappearing Flows Issues

### DISCUSSION THREAD

**Discussion by jcbgroup** [FlowiseAI/Flowise#570] [General]
I hope this message finds you well. I am currently facing significant issues with my Flowise application running on Render's platform and I am seeking your assistance.

The application in question is a Chatflow named "flowisetest". This is merely a test application where only two messages were sent for testing purposes. This flow is designed to be connected with Botpress, and the file serving the knowledge base is quite small, only 6.14kb in size.

Despite the minimal activity and small file size, my application has been running out of memory. This has been puzzling and concerning, as it could potentially impact the performance of my application and the service I provide to my clients. I have reached out to Render's support team, but they suggested that this issue might be at the application layer. Could you please provide any guidance or suggestions on how I might investigate this issue further?

Furthermore, I have recently discovered that my test "flowisetest" flow has disappeared from my application. Render's support team mentioned that they've heard a lot about flows disappearing and have reached out to you to update the docs. Could you please direct me to the specific part of the Flowise docs that address this issue? Or is there any other action you would recommend I take to prevent my flows from disappearing?

I appreciate your assistance and look forward to your response.

Best regards,
Ana Costa

**Comment by HenryHengZJ** [comment]
hey Ana, thanks for the details explanation. Here's the step by step guide on how to overcome the issue - https://docs.flowiseai.com/deployment/render#persistent-disk
